## Exponential Modeling

In [1]:
bacteria <- read.csv("bacteria.csv")

In [2]:
head(bacteria)

Period Count
1 0      15   
2 1      19   
3 2      21   
4 3      22   
5 4      28   
6 5      32

## Question Setup
We are trying to answer the question of how does much does bacteria grow over time. We will examine the change in Count, your y variable, over time Period, your x variable.

## Exponential Modeling Code

In [3]:
exMod <- lm(log(bacteria$Count)~bacteria$Period)
summary(exMod)


Call:
lm(formula = log(bacteria$Count) ~ bacteria$Period)

Residuals:
      Min        1Q    Median        3Q       Max 
-0.106253 -0.038496  0.006744  0.029930  0.077803 

Coefficients:
                Estimate Std. Error t value Pr(>|t|)    
(Intercept)     2.701306   0.027086   99.73  < 2e-16 ***
bacteria$Period 0.165330   0.003293   50.21 2.84e-16 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 0.0551 on 13 degrees of freedom
Multiple R-squared:  0.9949,	Adjusted R-squared:  0.9945 
F-statistic:  2521 on 1 and 13 DF,  p-value: 2.841e-16


## Interpretting the Output
By looking at the bottom F-statistic and associated p-value, you see that this model is significant! That means that this particular bacteria does demonstrate exponential growth over time. Looking at the Estimate column, you can see that for every one additional time b in, the bacteria has increased by 16%!